README.md

```diff
@@ -50,7 +50,8 @@ Returns a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Refe
 
   Key | Description | Default
   :--|:--|:--
-  `client` | [Client ID](https://developers.google.com/maps/documentation/javascript/get-api-key#specify-a-client-id-when-loading-the-api) | `undefined`
+  `channel` | [Client usage reporting channel](https://developers.google.com/maps/documentation/javascript/get-api-key#clientID-features) | `undefined`
```

Comment:
Consider changing the link to https://developers.google.com/maps/premium/reports/usage-reports#channels